backend/templates/create-report/template-files/routes/index.js

```diff
@@ -0,0 +1,21 @@
+const controller = require("@controllers/reports/$reportName.js");
+// const validators = require("@validators/report");
+const what = "report";
+
+const routes = [
+  {
```

Comment:
if we're just using a script to substitute ${reportName}, wouldn't it make more sense to abandon the Array, and simply assign routes an object instead?

that way the `registerRoutes` on Line 13 wouldn't need to iterate through an array of 1 items